charts/elastic-cloud/examples/multi-node.yaml

```diff
@@ -8,8 +9,10 @@ backups:
 
 kibana:
   enabled: true
+  namespace:
   ingress:
     enabled: false
+    namespace:
```

Comment:
should this ingress-level namespace inherit the namespace already given in `kibana.namespace`?

Reply:
Ahh yeah.  I'll update this for es/kibana